packages/lvm2/packaging

```diff
@@ -8,6 +8,14 @@ set -u # report the usage of uninitialized variables
 # $BOSH_INSTALL_TARGET - where you copy/install files to be included in package
 
 # We see lock issues, maybe we can check these?
+
+# Packages for Xenial
```

Comment:
Why do we keep all the packages in the VM? Why not check if it's xenial and copy selectively?

Reply:
made changes to copy only necessary packages based on ubuntu version

Reply:
thanks!